report_generator/outputs/full_report.md

```diff
@@ -0,0 +1,265 @@
+# Code report
+
+**Repository link:** https://repository.byont/
+
+**Commit number:** j230a-al4uf-zmh2
+
+**Relative path to contract in the repository being verified:** https://contract-path-url.byont/
+
+**Solidity Version:** 2.04
+
+**Foundry Version:** 0.85
+
+**Compiled Code Size:** 2345 KB
+
+**Notes from the developers:** Some notes about the whole testing, that will be manually written by the Byont developers
+
+**Number of vulnerabilities found per SWC:** 3465
+
+**Parameters per Tool:**
+
+
+
+
+
+## Slither
+
+**Total number of issues:** 42069
+
+**Version of the tool:** 1.54.33
+
+
+
+
+
+## reentrancy-eth
+
+* **Impact:** `High`
+* **Confidence:** `Medium`
+
+### In Function
+
+there should some sort of a code snapshot
+
+### Lines of Relevance
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+there also should be some sort of a code snapshot, but apparently it is usually a bit smaller than the ones for Functions
+
+
+
+
+
+
+
+
+## divide-before-multiply
```

Comment:
I think the structure name, impact, confidence, in function, lines of relevance is nice